src/Field/Group/AbstractGroupingElement.php

```diff
@@ -104,4 +104,26 @@ public function transformData (
 			$dataVisitor,
 		);
 	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function enableValidation (
+		bool $required = true,
+		?string $regexp = null,
+		bool $allowMissingData = false,
+	) : static
+	{
+		parent::enableValidation($required, $regexp, $allowMissingData);
+
+		foreach ($this->fieldCollection->getRootFields() as $rootField)
+		{
+			if ($rootField instanceof AbstractField)
+			{
+				$rootField->enableValidation($required, $regexp, $allowMissingData);
```

Comment:
All fields need to be abstract fields, so that check is always true.

Reply:
Unfortunately, the API is returning an `array<string, FieldDefinitionInterface>`, so I have to add this extra check, unless we're changing the internals.

Reply:
Ah yeah, you are right. Nevermind then. 👍